devices/src/virtio/mmio.rs

```diff
@@ -128,6 +128,12 @@ impl MmioDevice {
         })
     }
 
+    // Gets the encapsulated VirtioDevice
+    #[allow(clippy::borrowed_box)]
+    pub fn device_mut(&mut self) -> &mut Box<VirtioDevice> {
```

Comment:
Just wondering, is there any reason to return a `&mut Box<VirtioDevice>` instead of a `&mut dyn VirtioDevice` pointing at the box contents?

Reply:
Fixed